src/__tests__/TextEditor-spec.jsx

```diff
@@ -31,6 +31,21 @@ describe('TextEditor', () => {
        .not.toThrow();
   });
 
+  it('fires a change handler', () => {
+    const handleChange = jest.fn();
+    const wrapper = mount(
+      <TextEditor
+        value='test content'
+        onChange={handleChange}
+      />
+    );
+
+    expect(handleChange).not.toHaveBeenCalled();
+    const reactEl = wrapper.instance();
+    reactEl.handleEditorChange(wrapper.state('editorState'));
+    expect(handleChange).toHaveBeenCalled();
```

Comment:
@corescan Not sure if this test could be useful. We are manually triggering the change handler and then looking to see if it's called which will be always be correct! I think a better approach is to test if the textEditor values are getting modified properly per the result of onChange event.

Reply:
We have some logic that programmatically calls the changeHandler from within the component, so it's not the same as testing the HTML or React api, which we would assume is always correct. Another further test would be check that the changes took place as expected.